app/src/androidTest/java/org/simple/clinic/drugs/sync/PrescriptionSyncAndroidTest.kt

```diff
@@ -53,13 +54,16 @@ class PrescriptionSyncAndroidTest : BaseSyncCoordinatorAndroidTest<PrescribedDru
   val currentFacilityUuid: UUID
     get() = facilityRepository.currentFacilityUuid(user)!!
 
-  val authenticationRule = AuthenticationRule()
+  private val authenticationRule = AuthenticationRule()
+
+  private val registerPatientRule = RegisterPatientRule(UUID.randomUUID())
 
-  val registerPatientRule = RegisterPatientRule(UUID.randomUUID())
+  private val rxErrorsRule = RxErrorsRule()
 
   @get:Rule
   val ruleChain = RuleChain
       .outerRule(authenticationRule)
+      .around(rxErrorsRule)
```

Comment:
Should we make the `rxErrors` rule always the innermost one in the rule chain? It seems odd to break `authenticationRule` and `registerPatientRule`.

Reply:
Agreed. Fixing this.